src/compiler/emitter.ts

```diff
@@ -324,6 +324,7 @@ namespace ts {
                 case SyntaxKind.PrivateKeyword:
                 case SyntaxKind.ProtectedKeyword:
                 case SyntaxKind.PublicKeyword:
+                case SyntaxKind.OverrideKeyword:

```

Comment:
This has not been done yet.  I was not able to identify an appropriate declaration emitter test to mimic.  Can you suggest one to investigate?